cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -118,3 +118,67 @@
     end
   end
 end
+
+action :unmount do
+  shared_dir_array = new_resource.shared_dir_array.dup
+  vol_array = new_resource.vol_array.dup
+
+  shared_dir_array.each_with_index do |dir, index|
+    shared_dir_array[index] = dir.strip
+    shared_dir_array[index] = "/#{shared_dir_array[index]}" unless shared_dir_array[index].start_with?('/')
```

Comment:
This sanitization step for strings containing paths is executed in many places in our cookbook. 
What about using a dedicated function for it (in the part of code touched by this PR)?

Reply:
Good idea! Changed